### PR TITLE
Manage listing

### PR DESCRIPTION
**Pull request by vishalreddy9866**

### CODE REVIEW

File: frontend/src/components/items/ListItem.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            Cancel
+          </Button>
+          <Button variant="primary" type="submit">
+            Submit
+          </Button>
+        </div>
+      </Form>
+    </div>
+  );
+};
```
Comment (git-voo) [inline frontend/src/components/items/ListItem.js:74]: I think we may have to add more fields to collect category, location, and other parameters in the items schema

**Review by git-voo — APPROVED** [review]

File: frontend/src/components/navigation/index.js
Comment (shagunsharmaslu) [inline frontend/src/components/navigation/index.js:1]: Why we have statically defined categories and locations here. I think we can move this to DB and fetch from there?

File: frontend/src/styles/navBar/navbar.css
Comment (shagunsharmaslu) [inline frontend/src/styles/navBar/navbar.css:1]: can we if possible move all the buttons style to our generic button css under style folder?

**Review by nlp-saiteja — APPROVED** [review]
Lgtm!

**Review by git-voo — CHANGES_REQUESTED** [review]
This is bullshit.
Start over!!!!!!!

**Review by git-voo — COMMENTED** [review]
There's merge conflict on this branch that prevents us from merging to main. Could kindly review and resolve, please?

File: frontend/src/components/items/ListItem.js (hunk 1 above)
Comment (vishalreddy9866) [reply frontend/src/components/items/ListItem.js:235]: ive added these fields.

**Pull request merged**